### ISSUE TITLE
[BUG/ISSUE] Bug in implementation of LUO_WETDEP option in 13.2.0-beta.1

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#820]
Following the benchmarking of [GEOS-Chem 13.2.0](http://wiki.seas.harvard.edu/geos-chem/index.php/GEOS-Chem_13.2.0) with the [Luo et al. (2020)](https://gmd.copernicus.org/articles/13/2879/2020/) wet deposition scheme, Gan Luo noticed a bug in the code. @ganluoasrc wrote:

>Simulated sulfate mass concentration looks lower than what I expected.
>
>I checked https://github.com/geoschem/geos-chem/blob/13.2.0-beta.0/GeosCore/sulfate_mod.F90
>
>It seems that we missed some switches for Non-volatile aerosol concentration calculation:

```
(1) At line 5142 D = (2.e+0_fp*SO4nss) - (TNA+2.e+0_fp*TDCA), it should be:
#ifdef LUO_WETDEP
    D = (1.5e+0_fp*SO4nss) - (TNA+2.e+0_fp*TDCA)
#else
    D = (2.e+0_fp*SO4nss) - (TNA+2.e+0_fp*TDCA)
#endif

(2) At line 5156 D = (2.e+0_fp * SO4nss) - TNA), it should be:
#ifdef LUO_WETDEP
    D = (1.5e+0_fp * SO4nss) - TNA
#else
    D = (2.e+0_fp * SO4nss) - TNA
#endif
```

>Switching from 2.e+0_fp * SO4nss to 1.5e+0_fp * SO4nss is following the reasons introduced in Luo et al. (2020).

**Comment by msulprizio** [comment]
We will fix this and resubmit the 13.2.0 full-chemistry and transport tracer benchmarks with the `LUO_WETDEP` option enabled. The version with this fix will be tagged with `13.2.0-beta.2`.

**Comment by msulprizio** [comment]
This is now resolved in commit https://github.com/geoschem/geos-chem/commit/3d82b3a6ad2dcb0843e752616f7f761820a1a1aa (currently in `dev`)
